s3/pkg/datastore/gcp/gcp.go

```diff
@@ -136,7 +135,7 @@ func (ad *GcsAdapter) Get(ctx context.Context, object *pb.Object, start int64, e
 
 func (ad *GcsAdapter) Delete(ctx context.Context, input *pb.DeleteObjectInput) error {
 	bucket := ad.session.NewBucket()
-	objectId := input.Bucket + "/" + input.Key
+	objectId := input.Key
```

Comment:
Same here

Reply:
this is not relevant to PR but its a bug fix

Reply:
Plz update the PR description, mention what all fixes are included

Reply:
done